lib/laa_crime_schemas/types/types.rb

```diff
@@ -227,6 +227,19 @@ module Types
                               terraced
                               custom
                             ])
+
+    RelationshipType = String.enum(*%w[
+                                     business_associates
+                                     ex_partner
+                                     family_members
+                                     friends
+                                     house_builder
+                                     housing_association
+                                     local_authority
+                                     partner_with_a_contrary_interest
+                                     property_company
+                                     custom
```

Comment:
Is 'custom' equivalent to 'other'. If so, I wonder if better to use 'other' for consistency.